pymeasure/instruments/edwards/NXDS.py

```diff
@@ -0,0 +1,52 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2020 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+# from pymeasure.instruments import Instrument
+
+from time import sleep
+from pymeasure.instruments import Instrument
+
+class NXDS(Instrument):
+    """ Represents the Edwards nXDS (10i) Vacuum Pump
+    and provides a low-level interaction with the instrument.
+
+    This instrument is constructed to only start and stop pump.
+    """
+
+    def __init__(self, resourceName, **kwargs):
+        super(NXDS, self).__init__(
+            resourceName,
+            "Edwards NXDS Vacuum Pump",
+            **kwargs
+        )
+	
+    def start(self):
+        """ Starts pump with default settings.
+        """
+        self.write("!C802 1\r")
+		
+    def stop(self):
+        """ Starts pump with default settings.
+        """
+        self.write("!C802 0\r")
```

Comment:
To keep style consistent with `pymeasure` guidelines, I would replace these two functions with and `Instrument.setting` property (see also https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments.html#advanced-properties).
Eg. sometthing like:
```
enable = Instrument.setting("!C802 %d",
                              """ Command docs goes here""",
                              validator=strict_discrete_set,
                              values = (0,1),)
```
I think this could lay down an easy path to add also other instrument properties.

As a suggestion, I would add also 'Identification' and '?V802' query commands as properties.